diff --git a/dask/dataframe/accessor.py b/dask/dataframe/accessor.py
index 9a73eb6ce..54f72cd9a 100644
--- a/dask/dataframe/accessor.py
+++ b/dask/dataframe/accessor.py
@@ -1,5 +1,6 @@
 from __future__ import annotations
 
+import functools
 import warnings
 
 import numpy as np
@@ -28,8 +29,15 @@ def _bind_property(cls, pd_cls, attr, min_version=None):
 
     func.__name__ = attr
     func.__qualname__ = f"{cls.__name__}.{attr}"
+    original_prop = getattr(pd_cls, attr)
+    if isinstance(original_prop, property):
+        method = original_prop.fget
+    elif isinstance(original_prop, functools.cached_property):
+        method = original_prop.func
+    else:
+        method = original_prop
     try:
-        func.__wrapped__ = getattr(pd_cls, attr)
+        func.__wrapped__ = method
     except Exception:
         pass
     setattr(cls, attr, property(derived_from(pd_cls, version=min_version)(func)))
